website/src/documentation/patterns/side-inputs.md

```diff
@@ -50,25 +50,25 @@ For instance, the following code sample uses a `Map` to create a `DoFn`. The `Ma
 
 ## Slowly updating side input using windowing
 
-You can read side input pcollection periodically into distinct windows.
-Later, when you apply side input to your main input, windows will be matched automatically 1:1.
-This way, you can guarantee side input consistency on the duration of the single window.
-
-To do this, you can utilize PeriodicSequence PTransform that will generate infinite sequence
-of elements with some real-time period:
+You can read side input data periodically into distinct PCollection windows.
+Later, when you apply the side input to your main input, each main input
+window is automatically matched to a single side input window.
+This guarantees side input consistency on the duration of the single window,
+meaning that each window on main input side will be matched to a single
+version of side input data.
+
+To do this, you can utilize a combination of PeriodicSequence/PeriodicImpulse
```

Comment:
For formatting, it might be clearer to say "a combination of PeriodicSequence and PeriodicImpulse PTransforms" instead of "PeriodicSequence/PeriodicImpulse PTransforms" - just to make sure there's no ambiguity in regards to these being two separate transforms.